docs/third_party_packages.rst

```diff
@@ -150,6 +150,14 @@ for you.
         session.request(method=method, url=url, ...)
 
 
+Instead of having to pass ``serialize_with='prettyjson'`` each time, we can
```

Comment:
So this breaks up the narrative voice in this section. If we can move this after the literal include that has the pretty-printed JSON, that would be great. I would also prefer this to say

```rst
If we want to use this serializer for every request, we can make this a global default by adding this to our configuration:

.. code-block:: python

    config.default_cassette_options['serialize_with'] = 'prettyjson'
```